src/main/java/module/lineup/LineupPanel.java

```diff
@@ -145,82 +137,50 @@ private LineupRatingAssistantPanel getLineupRatingAssistantPanel() {
-				bar2.setValue(e.getValue());
-			}
-		};
-		bar.addAdjustmentListener(adjustmentListener);
-		bar2.addAdjustmentListener(adjustmentListener);
-
-		splitPane.add(scrollpane);
-		splitPane.add(scrollpane2);
-
-		return splitPane;
-	}
-
-	private void addListeners() {
-
-		ListSelectionListener lsl = new ListSelectionListener() {
-
-			@Override
-			public void valueChanged(ListSelectionEvent e) {
-				if (e.getSource() == lineupPlayersTable.getSelectionModel()) {
-					synchronizeSelection(lineupPlayersTable);
-				} else if (e.getSource() == lineupPlayersTableNameColumn.getSelectionModel()) {
-					synchronizeSelection(lineupPlayersTableNameColumn);
-				}
-			}
-
-			private void synchronizeSelection(JTable sourceTable) {
-				JTable targetTable;
-				if (sourceTable == lineupPlayersTable) {
-					targetTable = lineupPlayersTableNameColumn;
-				} else {
-					targetTable = lineupPlayersTable;
-				}
-
-				int row = sourceTable.getSelectedRow();
-				if (row == -1) {
-					targetTable.clearSelection();
-				} else {					
-					if (targetTable.getSelectedRow() != row) {
-						targetTable.setRowSelectionInterval(row, row);
-					}
-					Player player = ((PlayerTable) sourceTable).getPlayer(row);
-					if (player != null) {
-						HOMainFrame.instance().selectPlayer(player);
-					}
-				}
-			}
-		};
-
-		this.lineupPlayersTable.getSelectionModel().addListSelectionListener(lsl);
-		this.lineupPlayersTableNameColumn.getSelectionModel().addListSelectionListener(lsl);
-	}
+		return lineupPlayersTable;
+	}
+
+//	private void addListeners() {
+//
+//		ListSelectionListener lsl = new ListSelectionListener() {
+//
+//			@Override
+//			public void valueChanged(ListSelectionEvent e) {
+//				if (e.getSource() == lineupPlayersTable.getSelectionModel()) {
+//					synchronizeSelection(lineupPlayersTable);
+//				} else if (e.getSource() == lineupPlayersTableNameColumn.getSelectionModel()) {
+//					synchronizeSelection(lineupPlayersTableNameColumn);
+//				}
+//			}
+//
+//			private void synchronizeSelection(JTable sourceTable) {
+//				JTable targetTable;
+//				if (sourceTable == lineupPlayersTable) {
+//					targetTable = lineupPlayersTableNameColumn;
+//				} else {
+//					targetTable = lineupPlayersTable;
+//				}
+//
+//				int row = sourceTable.getSelectedRow();
+//				if (row == -1) {
+//					targetTable.clearSelection();
+//				} else {
+//					if (targetTable.getSelectedRow() != row) {
+//						targetTable.setRowSelectionInterval(row, row);
+//					}
+//					var tableModel = (PlayerOverviewTableModel)sourceTable.getModel();
+//					var player = tableModel.getPlayerAtRow(row);
+////					Player player = ((PlayerTable) sourceTable).getPlayer(row);
+//					if (player != null) {
+//						HOMainFrame.instance().selectPlayer(player);
+//					}
+//				}
+//			}
+//		};
+
+//		this.lineupPlayersTable.getSelectionModel().addListSelectionListener(lsl);
+//		this.lineupPlayersTableNameColumn.getSelectionModel().addListSelectionListener(lsl);
+//	}
```

Comment:
Just remove :)

Reply:
I will so so - just want to double check that i didn't miss any feature